java/src/test/java/com/adhoc/flight/TestAdhocFlightClient.java

```diff
@@ -107,6 +125,17 @@ public void testSimpleQuery() throws Exception {
         client.runQuery(SIMPLE_QUERY, true);
     }
 
+    @Test
+    @Ignore("Need to run flight server in encrypted mode.")
+    //TODO Enable encrypted flight server on actions.
```

Comment:
Should this still be here?

Reply:
I added because it was something that was being requested. However I disable the test because the github actions is not running an encrypted server. I can remove if you think is better